### DISCUSSION TITLE
What does MIN/MAX_QUORUM_VOTES_BPS mean in the NounsDAOLogicV1?

### DISCUSSION THREAD

**Discussion by cdt-eth** [nounsDAO/nouns-monorepo#413] [Solidity / smart contracts] [answered]
What does MIN/MAX_QUORUM_VOTES_BPS mean in the NounsDAOLogicV1?

**Comment by solimander** [comment, marked as answer]
MIN/MAX_QUORUM_VOTES_BPS are the vote quorum bounds as a percentage of the total Nouns supply, stored in basis points.

These values are currently set to 2% and 20%, respectively, which would require the vote quorum to be no less than 2% of the current Noun supply and no more than 20% of the Noun supply.
